lib/dockerregistry/inventory.go

```diff
@@ -1672,6 +1668,39 @@ func (rii *RegInvImage) ToYAML() string {
 	return b.String()
 }
 
+// ToFlattened is like ToYAML, but instead of printing things in an indented
+// format, it prints one image on each line as a CSV. If there is a tag pointing
+// to the image, then it is printed next to the image on the same line.
+//
+// E.g.
+//
+// nolint[lll]
+// a@sha256:0000000000000000000000000000000000000000000000000000000000000000,a:1.0
+// a@sha256:0000000000000000000000000000000000000000000000000000000000000000,a:latest
+// b@sha256:1111111111111111111111111111111111111111111111111111111111111111,-
+func (rii *RegInvImage) ToFlattened() string {
```

Comment:
`ToCSV` or `ToFlattenedCSV`?  Or `FormatAsFlattenedCSV` ?

It's also reasonable to pull this function into the "CLI layer" - it's essentially formatting it for the screen / stdout.

Reply:
I think `ToCSV` is probably best. Ack